src/avatar.cpp

```diff
@@ -814,9 +850,22 @@ void avatar::do_read( item &book )
         if( book_fun_for( book, *this ) != 0 ) {
             add_msg( m_info, _( "Reading this book affects your morale by %d" ), book_fun_for( book, *this ) );
         }
-        add_msg( m_info, ngettext( "A chapter of this book takes %d minute to read.",
-                                   "A chapter of this book takes %d minutes to read.", reading->time ),
-                 reading->time );
+
+        if( book.type->use_methods.count( "MA_MANUAL" ) ) {
+            const matype_id style_to_learn = martial_art_learned_from( *book.type );
+            add_msg( m_info, _( "You can learn %s style from it." ), style_to_learn->name );
+            add_msg( m_info, _( "This fighting style is %s to learn." ),
+                     martialart_difficulty( style_to_learn ) );
+            add_msg( m_info, _( "It would be easier to master if you'd have skill expertise in %s." ),
+                     style_to_learn->primary_skill->name() );
+            add_msg( m_info, ngettext( "Training session with this book takes %d minute.",
```

Comment:
```suggestion
            add_msg( m_info, ngettext( "A training session with this book takes %d minute.",
```

Reply:
To get a nice `std::string` for the time you could do 
```cpp
to_string( time_duration::from_turns( reading->time ) )
```
Maybe a little ugly looking but it would fix the minute vs minutes problem.